code/game/objects/items/stacks/tape.dm

```diff
@@ -27,6 +27,9 @@
 		to_chat(user, "<span class='warning'>[I] is already coated in [src]!</span>")
 		return
 
+	if(INTERACTING_WITH(user, I))
```

Comment:
don't do this, check in the do_after and just after the do_after to see if `I.embedding && I.embedding == conferred_embed` is set and bail out of the do after.

Reply:
ok but why, is it more foolproof in case more than one person is doing something or what.

Reply:
just cleaner than checking if the user is already interacting with the item